msi/GitProduct.wxs

```diff
@@ -70,6 +70,9 @@
         <Environment Name='PATH' Value='[BinFolder]' Action='set' Part='last' System='yes' />
         <RemoveFolder Directory='GitProgramMenuFolder' On='uninstall' />
       </Component>
+      <Component Directory="INSTALLFOLDER">
+        <RegistryValue Root="HKLM" Key="Software\GitForWindows" Name="CurrentVersion" Type="string" Value="$(var.Version)" />
+      </Component>
```

Comment:
Where is the corresponding change in `release.sh`? After reading the commit message, I thought that something would be moved out of that file, but nothing was...